flink-streaming-java/src/main/java/org/apache/flink/streaming/api/operators/StreamSource.java

```diff
@@ -154,7 +154,7 @@ public LatencyMarksEmitter(
 					public void onProcessingTime(long timestamp) throws Exception {
 						try {
 							// ProcessingTimeService callbacks are executed under the checkpointing lock
-							output.emitLatencyMarker(new LatencyMarker(timestamp, operatorId, subtaskIndex));
+							output.emitLatencyMarker(new LatencyMarker(System.currentTimeMillis(), operatorId, subtaskIndex));
```

Comment:
Thanks for your review, the test case has been reverted.

Reply:
I read the test case carefully, and found that this test case is use to verify ```SystemProcessingTimeService``` , but we don't accumulate a fixed time interval periodicity to producer millisecond time. so  the test logic is not useful, i remove this test again,if keep it,will cause BUILD FAILURE.

Reply:
> I read the test case carefully, and found that this test case is use to verify SystemProcessingTimeService

I'm curious how you arrived at that conclusion given that this class isn't used in this test.

The test ensures that sources emit the correct latency values, and as such has to stay. The test fails since it was based on the arbitrary timestamps, and not the actual processing time.
I've pushed an update to correct the test.

Reply:
>  The test fails since it was based on the arbitrary timestamps, and not the actual processing time.
you got my point.